src/app/app.module.ts

```diff
@@ -5,11 +5,12 @@ import { AppRoutingModule } from './app-routing.module';
 import { SharedModule } from './shared/shared.module';
 
 import { AppComponent } from './app.component';
-import { HeaderComponent } from './shared/components/header/header.component';
+import { LevelSelectModule } from './modules/level-select/level-select.module';
+import { SelectLevelComponent } from './modules/level-select/level-select.component';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule, SharedModule],
+  declarations: [AppComponent, SelectLevelComponent],
```

Comment:
Why can't you declare SelectLevelComponent in LevelSelectModule? It's even in that folder)

Reply:
Yes) it is my mistake, thank you.